fix(learn/css): add note that scroll bars are OS dependant (#34323)
mdn/content: files/en-us/learn/css/building_blocks/overflowing_content/index.md

**Remove some content from the box below. Notice how the scrollbars remain, even if there is no need for scrolling.**

> [!NOTE]

> > The scrollbar visibility also depends on the operating system.
> > You may have to change browser setting to always show scroll bars in order for following examples to work.

Reviewer: ```suggestion
> Scrollbar visibility depends on the operating system.
> You may have to change your browser settings to always show scroll bars in order for the scroll bars to always show in the following examples.
```
